packages/react/src/editor/BlockNoteView.tsx

```diff
@@ -139,27 +141,37 @@ function BlockNoteViewComponent<
     };
   }, [existingContext, editor]);
 
+  const setElementRenderer = useCallback(
+    (ref: (typeof editor)["elementRenderer"]) => {
+      editor.elementRenderer = ref;
+    },
+    [editor]
+  );
+
   return (
     <BlockNoteContext.Provider value={context as any}>
-      <EditorContent editor={editor}>
-        <div
-          className={mergeCSSClasses(
-            "bn-container",
-            editorColorScheme || "",
-            className || ""
-          )}
-          data-color-scheme={editorColorScheme}
-          {...rest}
-          ref={ref}>
+      <ElementRenderer ref={setElementRenderer} />
+      {!editor.headless && (
+        <EditorContent editor={editor}>
           <div
-            aria-autocomplete="list"
-            aria-haspopup="listbox"
-            ref={editor._tiptapEditor.mount}
-            {...contentEditableProps}
-          />
-          {renderChildren}
-        </div>
-      </EditorContent>
+            className={mergeCSSClasses(
+              "bn-container",
+              editorColorScheme || "",
+              className || ""
+            )}
+            data-color-scheme={editorColorScheme}
+            {...rest}
+            ref={ref}>
+            <div
+              aria-autocomplete="list"
+              aria-haspopup="listbox"
+              ref={editor.mount}
```

Comment:
@matthewlipski  because we changed this line, we needed to autobind the `mount` function (for `_tiptapEditor.mount` this was probably already the case)